### PR TITLE
add the crs version file to the repository

### PR DESCRIPTION
**Pull request by pnavarrc**
The CRS is a Google Spreadsheet that users can copy to create their own simulations. The spreadsheet includes a formula that fetches a file from an external source, parses its content, and, if the copy is outdated, shows a message to the user telling them to update to the new version.

Currently, the file with the version is hosted in https://raw.githubusercontent.com/Charles-CovidActNow/Covid-Response-Simulator/master/Covid%20Response%20Simulator

We will soon update the CRS to use a new version (see https://trello.com/c/o9TKhGdY/458-update-crs-link-version-125-https-docsgooglecom-spreadsheets-u-4-d-1vs-ojzbue96sky-8cnjwlpwnxkjwsdkw5xbejyr5-uk-copyuspsharing). 

This PR adds the file to our assets, so we can provide Gil with the URL of the version file. Once this PR is deployed, we will update the CRS URL and the version number.

### CODE REVIEW

**Review by mikelehen — COMMENTED** [review]
Hrm...  I don't think this PR will do anything, since the asset isn't imported by our code anywhere?  So it won't get compiled into our bundle or anything.

Perhaps you want to put it in `public/` instead so that it just gets served verbatim on the website (i.e. https://covidactnow.org/covid-response-simulator-version.txt)?

**Comment by pnavarrc** [comment]
Makes sense 😅  - updated. I might update the link URL as well once the spreadsheet is updated

File: src/screens/Resources/Resources.tsx [hunk 1]
```diff
@@ -20,6 +20,12 @@ import {
   ImageContainer,
 } from './Resources.style';
 
+/**
+ * When updating the COVID_RESPONSE_SIMULATOR_URL, update the version number
+ * and the URL in src/assets/covid-response-simulator-version.txt. This file is
```
Comment (mikelehen) [inline src/screens/Resources/Resources.tsx:25]: fix location here? 😁

**Review by mikelehen — APPROVED** [review]

File: src/screens/Resources/Resources.tsx (hunk 1 above)
Comment (pnavarrc) [reply src/screens/Resources/Resources.tsx:25]: updated, I just noticed!

**Comment by pnavarrc** [comment]
[Update CRS link: version 1.25 https://docs.google.com/spreadsheets/u/4/d/1vs-OjzBuE96sky-8CNJWLPWnXkJwsDKw5XBejYR5-uk/copy?usp=sharing](https://trello.com/c/o9TKhGdY/458-update-crs-link-version-125-https-docsgooglecom-spreadsheets-u-4-d-1vs-ojzbue96sky-8cnjwlpwnxkjwsdkw5xbejyr5-uk-copyuspsharing)

**Pull request merged**